docs/administration-guide/databases/aws-rds.md

```diff
@@ -0,0 +1,18 @@
+## Connecting to AWS's Relational Database Service (RDS)
+
+RDS offers several databases that Metabase officially supports, including PostgreSQL, MySQL, MariaDB, Oracle, and SQL server.
+
+Here's how to get connection information for databases on Amazon's RDS:
+
+1. Go to your AWS Management Console.
+   - Need help finding that? Visit [https://**My_AWS_Account_ID**.signin.aws.amazon.com/console](https://**My_AWS_Account_ID**.signin.aws.amazon.com/console). Be sure to insert your own AWS Account ID, though!
+2. Under **Database** services, click **RDS**.
+3. Then click **Instances**.
+4. Select the database you want to connect to Metabase.
+5. Write down the following information based on your database:
```

Comment:
> Write down

This sounds like you're intstructing them to copy the information from the RDS console view so that they can copy it over to Metabase, right? Seems a little goofy. "Type in the following information" or something instead?